### PR TITLE
[CELEBORN-653][TEST] Fix invalid configuration key in SparkTestBase

### PR DESCRIPTION
**Pull request by pan3793**
### What changes were proposed in this pull request?

Dot is missing after `spark`

### Why are the changes needed?

Correct the configuration key.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Pass GA.

### CODE REVIEW

File: tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala [hunk 1]
```diff
@@ -45,20 +45,16 @@ trait SparkTestBase extends AnyFunSuite
     shutdownMiniCluster()
   }
 
-  def updateSparkConf(sparkConf: SparkConf, sort: Boolean): SparkConf = {
+  def updateSparkConf(sparkConf: SparkConf, shuffleWriteMode: String): SparkConf = {
```
Comment (zwangsheng) [inline tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala:48]: How about using `ShuffleMode` enum for args type?

**Review by zwangsheng — APPROVED** [review]
LGTM + 1, with minor comments

File: tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala (hunk 1 above)
Comment (pan3793) [reply tests/spark-it/src/test/scala/org/apache/celeborn/tests/spark/SparkTestBase.scala]: Thanks, changed

**Comment by pan3793** [comment]
Merged to main/0.3

**Pull request closed without merging**